src/phoenix/datasets/experiments.py

```diff
@@ -24,6 +24,7 @@
     get_env_host,
     get_env_port,
 )
+from phoenix.datasets.tracing import trace_stealing
```

Comment:
lol, we might want to find a better word here.

Reply:
I think capture might be a less scary word?

Reply:
grand theft instrumentation